message.go

```diff
@@ -269,6 +273,70 @@ func (c *Client) UpdateMessage(msg *Message, msgID string) (*Message, error) {
 	return resp.Message, nil
 }
 
+// PartialUpdateMessage partially updates message with given msgID.
+func (c *Client) PartialUpdateMessage(msgID string, updates PartialUpdate, options map[string]interface{}) (*Message, error) {
+	switch {
+	case len(updates.Set) == 0 && len(updates.Unset) == 0:
+		return nil, errors.New("updates is empty")
```

Comment:
`errors.New("updates are empty")`  or `errors.New("updates parameter is empty")`